### ISSUE TITLE
Action bars cleared on load in of toon

### ISSUE THREAD

**Issue by Hundrednails** [javajd17wow/ajcTracker#7]
Add popup before AJC load in stating that your action bars will be reset entirely if moving forward with AJCmacros

**Comment by javajd17wow** [comment]
Agreed action upon discussion was to present a preemptive Popup warning them continuing with AJC will reset all of their macroassignments.
